### PR TITLE
Added systemd timers doc

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski

### CODE REVIEW

File: os/task-scheduler-with-systemd-timers.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Task scheduler with systemd timers
```
Comment (robszumski) [inline os/task-scheduler-with-systemd-timers.md]: title case

what about: "Scheduling Tasks with systemd Timers"?

File: os/task-scheduler-with-systemd-timers.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+CoreOS doesn't use `cron` scheduling daemon. The replacement is `systemd` timer unit. Here we will show you how you can schedule a periodic job.
```
Comment (robszumski) [inline os/task-scheduler-with-systemd-timers.md]: I feel like this leads off with a negative. Should we say "CoreOS uses systemd timers to..." and then say that its similar to cron, but with better syntax.

File: os/task-scheduler-with-systemd-timers.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+```
+[Unit]
+Description=Prints date into /tmp/date file
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/sh -c '/usr/bin/date >> /tmp/date'
+```
+
+Then we have to create timer unit `/etc/systemd/system/date.timer`:
```
Comment (robszumski) [inline os/task-scheduler-with-systemd-timers.md]: These have to be named the same, correct? If so, we need to mention that.

File: os/task-scheduler-with-systemd-timers.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+```
+[Unit]
+Description=Run date.service every 10 minutes
+
+[Timer]
+OnCalendar=*:0/10
+```
+
+This config will run `date.service` every 10 minutes. You can also list all timers enabled in your system using `systemctl list-timers` command or `systemctl list-timers --all` to list all timers. Run `systemctl start date.timer` to enable timer.
```
Comment (robszumski) [inline os/task-scheduler-with-systemd-timers.md]: Nice job mentioning all this stuff!

File: os/task-scheduler-with-systemd-timers.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## Cloud Config
```
Comment (robszumski) [inline os/task-scheduler-with-systemd-timers.md]: Cloud-Config

**Comment by kayrus** [comment]
Done!

**Comment by robszumski** [comment]
Nice looks great. This is cool, I didn't realize this:

> You can also create timer with different name, i.e. `task.timer`. In this case you have specify service unit name:

**Pull request merged**